src/sys/signal.rs

```diff
@@ -414,7 +414,7 @@ cfg_if! {
     if #[cfg(target_os = "redox")] {
         type SaFlags_t = libc::c_ulong;
     } else if #[cfg(target_env = "uclibc")] {
-        type SaFlags_t = libc::c_ulong;
```

Comment:
From the comment you linked, this change should be done to `#[cfg(all(target_env = "uclibc", target_arch = "mips*"))]`, why are we doing this for all `uclibc` triples?🤔

Reply:
Though if we do this:

>  this change should be done to #[cfg(all(target_env = "uclibc", target_arch = "mips*"))]

We need that libc PR and change  our libc dependency from the `libc-0.2` branch to `main`, as they are breaking changes for the libc crate.